runtime-modules/common/src/council.rs

```diff
@@ -1,12 +1,29 @@
 use frame_support::dispatch::DispatchResult;
+use sp_arithmetic::traits::Saturating;
 
 /// Provides an interface for the council budget.
-pub trait CouncilBudgetManager<Balance> {
+pub trait CouncilBudgetManager<AccountId, Balance: Saturating> {
     /// Returns the current council balance.
     fn get_budget() -> Balance;
 
     /// Set the current budget value.
     fn set_budget(budget: Balance);
+
+    /// Remove some balance from the council budget and transfer it to the account. Fallible.
+    fn try_transfer(account_id: &AccountId, amount: Balance) -> DispatchResult;
+
+    /// Remove some balance from the council budget and transfer it to the account. Infallible.
+    fn transfer(account_id: &AccountId, amount: Balance) {
```

Comment:
`withdraw` is a bit more explicit

Reply:
Can you also explicitly say that if the budget does not cover full withdrawal, there is no side-effect.

Reply:
Addressed both comments.